og.module

```diff
@@ -86,6 +86,16 @@ define('OG_MEMBERSHIP_REQUEST_FIELD', 'og_membership_request');
  */
 
 /**
+* Implements hook_entity_field_storage_info_alter().
+*/
+function og_entity_field_storage_info_alter(&$fields, \Drupal\Core\Entity\EntityTypeInterface $entity_type) {
+  if (!empty($fields[OG_AUDIENCE_FIELD])) {
```

Comment:
note that `OG_AUDIENCE_FIELD` is just the default name of the fields - There could be multiple group audience fields, which we currently recognize by the selection handler's name (it should be prefixed by `og_`).

Use can use [this](https://github.com/amitaibu/og/blob/8.x-1.x/og.module#L1847-L1850) function, which should probably move to the `Og` class